google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -77,6 +79,8 @@ interfaces:
         - topic
         - push_config
         - ack_deadline_seconds
+        - retain_acked_messages
+        - message_retention_duration
```

Comment:
It's not necessary for all fields to be in a flattening group. The point of a flattening group is to serve use cases which are both common and have a short list of parameters. Adding these two makes it a long list. I would just recommend not adding them; in Java, users can use the request-based method to access all parameters.